hm_pyhelper/tests/test_diagnostic_report.py

```diff
@@ -1,8 +1,8 @@
 import unittest
 import json
 
-from hm_pyhelper.diagnostics import Diagnostic
-from hm_pyhelper.diagnostics import DiagnosticsReport
+from hm_pyhelper.diagnostics.diagnostic import Diagnostic
```

Comment:
I'm surprised this is necessary given the way `diagnostics/__init__.py` is setup.